scripts/setUpSite.js

```diff
@@ -15,17 +15,42 @@ const base = new TileLayer(
   }
 );
 
+const setUpCityPointsLayer = async (map) => {
+  const scope = {
+    Regional: "#7b3294",
+    "City Center": "#fdae61",
+    Citywide: "#d7191c",
+    "Main Street": "#abdda4",
+    TOD: "#2b83ba",
+  };
+  data.forEach((mandate) => {
+    new CircleMarker([mandate.lat, mandate.long], {
+      // add layer id
+      radius: 7,
+      stroke: true,
+      weight: 0.9,
+      color: "#FFFFFF",
+      fillColor: scope[mandate.magnitude_encoded],
+      fillOpacity: 1,
+      // add label
+      // add options
+    }).addTo(map);
+  });
+};
+
 const setUpSite = async () => {
   setUpIcons();
 
   const map = new Map("map", {
     layers: [base],
   });
-  map.setView([40.9437, -78.9709], 12); // latitude, longitude
+  map.setView([43.2796758, -96.7449732], 4); // latitude, longitude
```

Comment:
This comment isn't very helpful. Better would be something like "Set default view to the United States"